docs/csharp/fundamentals/tutorials/pattern-matching.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: "Tutorial: Build algorithms with pattern matching"
 description: This advanced tutorial demonstrates how to use pattern matching techniques to create functionality using data and algorithms that are created separately.
-ms.date: 10/06/2020
+ms.date: 10/01/2021
 ms.custom: contperf-fy21q1
 ---
 # Tutorial: Use pattern matching to build type-driven and data-driven algorithms
 
-C# 7 introduced basic pattern matching features. Those features are extended in C# 8 and C# 9 with new expressions and patterns. You can write functionality that behaves as though you extended types that may be in other libraries. Another use for patterns is to create functionality your application requires that isn't a fundamental feature of the type being extended.
+C# 7 introduced basic pattern matching features. Those features are extended in C# 8 thru C# 10 with new expressions and patterns. You can write functionality that behaves as though you extended types that may be in other libraries. Another use for patterns is to create functionality your application requires that isn't a fundamental feature of the type being extended.
```

Comment:
Honest Q, not being snarky: Is *thru* an accepted spelling? I'd always assumed that it was considered grammatically incorrect, but times do change. And we all know spelling it *through* was just some Englishman's attempt to cheat at Scrabble.

Reply:
You're right. My bad.